modules/Microsoft.Network/virtualHubs/deploy.bicep

```diff
@@ -93,7 +93,7 @@ resource defaultTelemetry 'Microsoft.Resources/deployments@2021-04-01' = if (ena
   }
 }
 
-resource virtualHub 'Microsoft.Network/virtualHubs@2021-08-01' = {
+resource virtualHub 'Microsoft.Network/virtualHubs@2022-05-01' = {
```

Comment:
@aavdberg can you please rerun the readme generator script locally to update the current documentation for the module. It's located here 
https://github.com/Azure/ResourceModules/blob/main/utilities/tools/Set-ModuleReadMe.ps1

 Readme guidelines are also available in the wiki: https://github.com/Azure/ResourceModules/wiki/Contribution%20guide%20-%20Generate%20module%20Readme

Also please rerun the module pipeline and attach the result to the PR

Reply:
I tried to run the Set-ModuleReadMe.ps1 but loading the PowerShell file does not load the functions because of the Required 6.0 I think, and the example for the ResourceTypes is also not truly clear. Could you explain what the precies command is for this module?

Reply:
@ahmadabdalla  I tried to run the Set-ModuleReadMe.ps1 but loading the PowerShell file does not load the functions because of the Required 6.0 I think, and the example for the ResourceTypes is also not truly clear. Could you explain what the precies command is for this module?

Reply:
@aavdberg Try something like this

```powershell

(Get-ChildItem - Path "$pathToRepository\modules\Microsoft.Network\virtualHubs" -Recurse Filter 'deploy.bicep') .FullName | ForEach-Object { Set-ModuleReadMe -TemplateFilePath $_ -error Action 'Continue'}

```

You still need to load the function first.

Also please update the child modules to use the latest API as well:
- hubVirtualNetworkConnections
- hubRouteTables

Reply:
Have updated the other ones also the API versions and also updated the readme's with the correct version.

Reply:
@aavdberg I've replayed your branch locally and there's additional files you need to update in the readme. Running the script on your branch generated the following change requirement:

![image](https://user-images.githubusercontent.com/28486158/198745501-fb7c6b73-149d-4d12-b65f-53abc4703d29.png)

Can you please update the documentation and attach a successful deployment status badge against the updated module:

![image](https://user-images.githubusercontent.com/28486158/198745772-724580b9-53ef-4a51-997f-f7c663610a60.png)

Reply:
Updated